packages/coil-extension/src/background/services/AuthService.ts

```diff
@@ -228,11 +228,14 @@ export class AuthService extends EventEmitter {
         inTippingBeta: featureFlagResp.data.featureEnabled,
         minimumTipLimit: minTipLimitResp.data.minTipLimit.minTipLimit
           ? Number(minTipLimitResp.data.minTipLimit.minTipLimit) / 100
-          : 1,
+          : 1, // convert from cents to dollars
         remainingDailyAmount: this.store.user?.tipping?.limitRemaining
           ? Number(this.store.user?.tipping?.limitRemaining) / 100
           : 0, // convert from cents to dollars
-        hotkeyTipAmounts: [5, 10, 50] // dollar amounts - not yet set by user
+        lastTippedAmountUSD: this.store.user?.tipping?.limitRemaining
```

Comment:
nit: This math should really be a share util function since we are doing it all over the place.

Reply:
I actually really want to remove this section and do it all better, I just don't know extactly what I wanted to do just yet. I figured I would match what I had there for now and make the refactor on a new branch.